script.js

```diff
@@ -21,6 +28,32 @@ class Book {
     Book.prototype.saveTolocal();
   }
 
+  static showDate() {
+    const current = new Date();
+    const month = [];
+    month[0] = 'January';
+    month[1] = 'February';
+    month[2] = 'March';
+    month[3] = 'April';
+    month[4] = 'May';
+    month[5] = 'June';
+    month[6] = 'July';
+    month[7] = 'August';
+    month[8] = 'September';
+    month[9] = 'October';
+    month[10] = 'November';
+    month[11] = 'December';
+
+    const cDate = `${month[current.getMonth()]} ${current.getDate()}th ${current.getFullYear()},`;
+    const cTime = `${current.getHours()}:${current.getMinutes()}:${current.getSeconds()}`;
+    const dateTime = `${cDate} ${cTime}`;
+
+    const myDate = document.querySelector('#date');
+    const showMyDate = document.createElement('div');
+    showMyDate.innerHTML = `${dateTime}`;
+    myDate.appendChild(showMyDate);
+  }
```

Comment:
- [ ] The current date is displayed below the navigation bar, for this requirement, students should use the  [Luxon](https://moment.github.io/luxon/api-docs/index.html) library. See Get the current date and time
=> This requirement is not fully implemented, the date and time is present but you have to use [Luxon](https://moment.github.io/luxon/api-docs/index.html) as it was required in project requirement, try that out so that it will help you learn more about different libararies